src/commands/workflows/connections/disconnect.ts

```diff
@@ -36,7 +36,7 @@ export default class Workflows extends BaseCommand<typeof Workflows> {
       )
       const integrationsForConnectionId = await getIntegrationsForConnection(tenantId, selectedConnection.id)
       const choices = integrationsForConnectionId.data.map((x) => {
-        return {name: `[Type: ${x.integration_type}] in ${x.org_id} (integr ${x.id})`, value: x.id}
+        return {name: `[Type: ${x.integration_type}] in org ${x.org_id} (integration ${x.id})`, value: x.id}
```

Comment:
q: what's the difference in output here?

Reply:
it indicates the uuid displayed is for the org ID and spells out integration in full in the multiple selection step. it's to help understanding what uuid corresponds to what